### DISCUSSION TITLE
Separate lists from Favorites – Think Reddit’s multireddit

### DISCUSSION THREAD

**Discussion by TFWol** [community/community#9371] [Other Feature Feedback, Questions, & Ideas]
I don’t want items in a List to pollute my stared repos (which I sort of treat as favorites). 

Suggestion:
Along the same lines how #8702 put it: Separate out the features.
It'll be easier for you to work on suggestions and you won't inadvertently break the Starring feature everyone's been using for years.
It also just makes sense.

### Idea
It can be done similar to Reddit’s **multireddit**; you don’t _have_ to join a subreddit to put it into a multireddit, but you can if you want.

You shouldn’t need to Star a Repo to put it into a List, but you should be able to if you want.

**Comment by brianlovin** [comment]
Thanks for the note @thefinalwolf — we're going to be keeping a close eye on feedback like this to figure out how closely linked Lists and Stars should be. Could you share more details about what your use case is where you'd want to add a repository to a list, but _not_ star it?

**Reply by TFWol** [reply]
Absolutely, I can think of a few, but one example is say a user I follow has a ton of very interesting repos I might want to check out some time, so I could add items I would like to view someday by putting them in one or more lists such as "**To check out - Cyber**", "**To check out - Automation**" or even a dislike category "**Don't bother**" so I know I've already checked it out.
Mostly personal stuff and it really depends on the features. Maybe even have the option to make particular lists public.

The starred repos I would like to treat as favorites, the stuff I don't mind other users seeing me follow and don't mind having up front and center. It makes the stars I have more meaningful.
Having the option to also put those in lists, even side-by-side non-starred items, could help for categorizing them further.

I would put things in something like a Gist, but the ability to click just a couple of times to add items to lists is super convenient, especially while on the go.
In general, having the lists fairly malleable allows for more use cases.

**Reply by brianlovin** [reply]
Gotcha, thanks for the extra context!
